lib/action_network_rest/people.rb

```diff
@@ -19,5 +19,37 @@ def unsubscribe(id)
       response = client.put_request "#{base_path}#{url_escape(id)}", request_body
       object_from_response(response)
     end
+
+    def find_id_by_email(email)
+      url_encoded_filter_string = url_escape("email_address eq '#{email}'")
+      response = client.get_request "#{base_path}?filter=#{url_encoded_filter_string}"
+      person_action_network_id_from_query_response(response)
+    end
+
+    private
+
+    def person_action_network_id_from_query_response(response)
+      # This works for parsing exactly 1 person's info out of the response.
+      # The response we get from Action Network is expected to have
+      #
+      # "_embedded": {
+      #   "osdi:people": [{
+      #       "identifiers": [
+      #           "action_network:c947bcd0-929e-11e3-a2e9-12313d316c29"
+      #            ....
+      #        ]
+      #    }]
+      # }
+      #
+      # If so, we pull out the action_network identifier and return it
+      identifiers = response.body[:_embedded]['osdi:people'].first[:identifiers] || []
```

Comment:
Can we DRY this with [`object_from_response`](https://github.com/controlshift/action-network-rest/blob/master/lib/action_network_rest/base.rb#L14) in `Base`? (perhaps by pulling out the common ID-extraction code into another method)